### PR TITLE
various UI updates

### PR DESCRIPTION
**Pull request by douglasbakkum**
(easier if first merge #7 )

Description of changes are in the commit titles.
Contains a number of subjective style changes that can be discussed.

screen shot 2016-05-21 at 23 13 38
screen shot 2016-05-21 at 23 57 54
screen shot 2016-05-22 at 13 28 20
screen shot 2016-05-22 at 00 05 10
screen shot 2016-05-22 at 01 22 08
screen shot 2016-05-22 at 13 17 18

### CODE REVIEW

**Comment by jonasschnelli** [comment]
utACK.
Awesome work.
Will test and merge soon.

File: src/qt/dbb_gui.cpp [hunk 1]
```diff
@@ -2081,63 +2081,97 @@ void DBBDaemonGui::updateTransactionTable(DBBWallet *wallet, bool historyAvailab
...
         {
             QStandardItem *item = new QStandardItem(QString::fromStdString(txidUV.get_str()) );
-            transactionTableModel->setItem(cnt, 4, item);
+            transactionTableModel->setItem(cnt, 0, item);
         }
 
         cnt++;
     }
+  
+    if (!cnt) {
```
Comment (jonasschnelli) [inline src/qt/dbb_gui.cpp:2164]: Hmm... not sure if we should hide the table. IMO a empty table will show users that there will be a listing of transaction even if they have not done any yet.

Comment (douglasbakkum) [inline src/qt/dbb_gui.cpp:2164]: I agree. After playing with it a while, I agree it would be better to show an empty table. I'll update the PR.

**Comment by jonasschnelli** [comment]
Closing. Mostly merged upstream.

**Pull request closed without merging**